ra/ra.go

```diff
@@ -2365,6 +2365,29 @@ func (ra *RegistrationAuthorityImpl) checkOrderNames(names []string) error {
 	return nil
 }
 
+// GenerateOCSP looks up a certificate's status, then requests a signed OCSP
+// response for it from the CA. If the certificate status is not available
+// or the certificate is expired, it returns berrors.NotFoundError.
```

Comment:
Possibly worth explicitly noting that this does not save the newly-generated OCSP response back to the certificateStatus table.

Reply:
Done!